src/sonic-yang-mgmt/tests/test_cfghelp.py

```diff
@@ -126,8 +126,8 @@ def setUp(self):
         self.script_file = os.path.join(self.test_dir, '..', 'sonic-cfg-help')
 
     def run_script(self, argument):
-        print('\n    Running sonic-cfg-help ' + argument)
-        output = subprocess.check_output(self.script_file + ' ' + argument, shell=True)
+        print('\n    Running sonic-cfg-help ', argument)
```

Comment:
>, [](http://example.com/codeflow?start=45&length=1)

Semgrep alerts on this line?

Reply:
No, but `argument` variable is a list, cannot be concatenated with string in the print statement.

Reply:
```suggestion
        print('\n    Running sonic-cfg-help ' + shlex.join(argument))
```

Reply:
Updated.